src/components/DirectPaymentConfirmation/DirectPaymentConfirmation.tsx

```diff
@@ -0,0 +1,88 @@
+import { Notification } from "@stellar/design-system";
+
+import { CreateDirectPaymentRequest, ApiReceiver } from "types";
+import { shortenAccountKey } from "helpers/shortenAccountKey";
+
+import "./styles.scss";
+
+interface DirectPaymentConfirmationProps {
+  paymentData: CreateDirectPaymentRequest;
+  selectedReceiver: ApiReceiver | null;
+  receiverSearch: string;
+}
+
+interface ConfirmationRowProps {
+  label: string;
+  children: React.ReactNode;
+}
+
+const ConfirmationRow: React.FC<ConfirmationRowProps> = ({ label, children }) => (
+  <div className="DirectPaymentConfirmation__row">
+    <span className="DirectPaymentConfirmation__label">{label}</span>
+    <span className="DirectPaymentConfirmation__value">{children}</span>
+  </div>
+);
+
+export const DirectPaymentConfirmation: React.FC<DirectPaymentConfirmationProps> = ({
+  paymentData,
+  selectedReceiver,
+  receiverSearch,
```

Comment:
Something is off with the search. We used to get the response back right away, but now it takes a long time and isn't consistent. Could you please take a look? It might be something to do with how it's passed as a prop. Maybe the child component isn't updating on every value change from the parent.